experiments/mp-game-6/draw.creatures.js

```diff
@@ -210,3 +225,25 @@ function markAsSpecies(id) {
 function unmarkAsSpecies(id) {
     $(id).css({"background-color":"transparent"});
 }
+function color(num){
```

Comment:
Unless you rely on the `default` case often, an array might be a faster way of doing this

```javascript
var colors = ['red', 'orange', 'yellow', 'green', 'blue'];
var a_color = colors[4];
```